LaMulanaItemTrackerForm.Designer.cs

```diff
@@ -921,7 +917,6 @@ private void InitializeComponent()
             this.mantra.Location = new System.Drawing.Point(52, 452);
             this.mantra.Name = "mantra";
             this.mantra.Size = new System.Drawing.Size(40, 40);
-            this.mantra.SizeMode = System.Windows.Forms.PictureBoxSizeMode.CenterImage;
```

Comment:
I think this one was centered on purpose, because I had a smaller-size djed pillar on top of the mantra background image. Or is that a different config?

Same thing also applies to maps (with the dragon bone)